### PR TITLE
Update node-providers.md

### PR DESCRIPTION
**Pull request by yunseonna**
Proposing to update additional possible free RPC endpoint powered by All That Node.

**What changed? Why?**

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

**Review by wbnns — CHANGES_REQUESTED** [review]
@yunseonna 

Hi there! Looks like the link for All That Node goes to a page about zkSync -- is there a Base-specific landing page you can update the link to? Otherwise, this would be confusing to Base users.

**Comment by yunseonna** [comment]
Oh sorry, I've changed the link!

File: apps/base-docs/docs/tools/node-providers.md [hunk 1]
```diff
@@ -62,6 +62,17 @@ hide_table_of_contents: true
 
 ---
 
+## All That Node
+
+[All That Node](https://www.allthatnode.com/base.dsrv) is the most comprehensive multi-chain development suite, designed to support multiple networks from a single platform. They offer free and [paid plans](https://www.allthatnode.com/pricing.dsrv) with additional features and increased request limits.
```
Comment (wbnns) [inline apps/base-docs/docs/tools/node-providers.md:67]: @yunseonna 

Could you also please update to remove the superlative?

s/is the most/is a/

**Review by wbnns — CHANGES_REQUESTED** [review]
Left a small nit in-line.

**Comment by yunseonna** [comment]
I've chenaged it! Thanks in advance.

**Review by wbnns — APPROVED** [review]
ACK

@yunseonna Thanks for the updates and for building on Base 🔵

**Pull request merged**